Added landing page features and respective overview docs (#10)
Ammalgam-Protocol/docs: docs/overview.md

borrow against. Additionally, we built a mechanism to allow for borrow AMM
shares themselves which can be used as a hedge for impermanent loss or to make
straddles. The feedback we have gotten from sophisticated traders in the space
is extremely positive because what we are building is so powerful while our UI
makes the complexity intuitive and accessible.

### Trading

Ammalgam was forked from Uniswap v2 and thus has parity with any DEX.
Although we don't support concentrated liquidity, we offer similar functionality
in the form of leverage. Users can borrow the underlying x and y in the pair
against their market making position allowing for the creation of larger
positions that what is possible with their own assets. This gives the exact same
payout or risk of a concentrated liquidity position in while it is in range, but
also allows for the position to continue to earn swap fees when the price leaves
the range. This also requires users have to monitor the price and its effect on
their position since it can be liquidated if the impermanent loss from price
change diminishes the value such that their debt is no longer has sufficient
collateral to secure it. 

### Lending

> We built a over collateralized lending protocol into the Uniswap v2 pair

Reviewer: ```suggestion
We built a overcollateralized lending protocol into the Uniswap v2 pair
```

contract. This means that the two assets in the pair contract can be borrowed
and deposited to earn lending yield but not be included in the reserves used to
quote swaps. 

Additionally, we allow users to borrow liquidity or market maker shares to give
exposure to positive gamma, or "impermanent gain". This is done by allowing the
underlying x and y in the pair to be borrowed together in proportion to the
trading reserves and tracking the debt in units of $L$, or $L = \sqrt{k} =
\sqrt{x + y}$. This means that debt in $L$ are static, but the underlying assets
$x$ and $y$ will depend on the price. As an asset increases in value, a borrower
will owe less of the asset that increased in value and more of the asset that
decreased in value.

### Capital Efficiency

> Market making using uniswap v2 is considered inefficient because most of the

Reviewer: ```suggestion
Market making using Uniswap v2 is considered inefficient because most of the
```

makers, but also increased the rate at which a market makers position lost value
as the price changed. The v3 solution was in fact solving a problem by
leveraging the underlying inefficiency.

Ammalgam rethinks this problem from a different angle. What if we could build 
what Hayden suggests at the protocol level? 

<Tweet id="1452832342788169732" />

This is what Ammalgam does, it lends out the excess assets in the Uniswap v2
pair. If that is not enough by itself, this efficiency can be further leveraged
for those who understand the risks and rewards of doing so.

### Utility (δ, γ)

Ammalgam recomposes DeFi into one protocol! With both trading and lending as
native features of our protocol, we unlock a massive amount of utility not
currently accessible in DeFi in one protocol. We can create a mind blowing
amount of uses from this combination that can be described in two variables
familiar to options traders, delta and gamma. 

Delta gives users the ability to go long or short by borrowing an asset and
selling it for the other. Gamma describes the amount of exposure to market
making, including negative exposure by borrowing market maker shares. The

> payout charts accessible can be seen in the desmos chart below. 

Reviewer: ```suggestion
payout charts accessible can be seen in the Desmos chart below. 
```

<iframe 
  src="https://www.desmos.com/calculator/zzgneljqca"
  frameBorder="0" 
  allowFullScreen
  width="100%"
  height="600">
</iframe>

#### Recipes

We also have tried to simplify this somewhat overwhelming range of positions by
creating a set of "recipes". 

- Market making
- Short or Long
- Leveraged Market Making
- Delta Neutral Market Making
- Short or Long Market Making
- Straddle
- Call or Put

> #### Payout charts

Reviewer: ```suggestion
#### Payout Chart
```

When selecting a recipe or configuring a custom strategy with (δ, γ), our UI
shows the respective payout chart for that position giving users immediate
feedback of the effect of their changes. 

> #### Heat map

Reviewer: ```suggestion
#### Heat Map
```

Additionally we have built a visualization tool that shows the impact of both
price change and the fees earned or paid over time. The heat map shows the
positions value after any given period of time and price change by representing
position value as a color (heat). Time is the horizontal axis and price change
is the vertical access. Below is an example of the heat map for the delta

> neutral market making

Reviewer: ```suggestion
Delta Neutral Market Making
```

Author: done

> is the vertical access. Below is an example of the heat map for the delta
> neutral market making

Author: ```suggestion
is the vertical access. Below is an example of the heat map for the Delta
Neutral Market Making
```
